.changes/unreleased/Features-20230912-153013.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
+body: Allow setting access in config
```

Comment:
Can you make this a bit more clear so that someone reading it without context will have a better idea what it means?